pkg/centralsensor/sensor_hello.go

```diff
@@ -39,3 +40,8 @@ func DeriveSensorHelloFromIncomingMetadata(md metautils.MD) (*central.SensorHell
 	sensorHello.Capabilities = sliceutils.StringSlice(extractCapsFromMD(md).AsSlice()...)
 	return sensorHello, versionErr
 }
+
+func SecuredClusterIsNotManagedManually(helmManagedConfig *central.HelmManagedConfigInit) bool {
```

Comment:
[nit] I'm not sure why the linter is not catching it but I think we want to have docs for all the exported functions.